src/components/dialog/index.tsx

```diff
@@ -171,8 +172,8 @@ class Dialog extends Component<DialogProps, DialogState> {
     }
   }
 
-  onDismiss = () => {
-    this.setState({modalVisibility: false}, () => {
+  _onDismiss = () => {
+    this.setState({modalVisibility: false, fadeOut: false}, () => {
```

Comment:
I didn't completely understand the `fadeOut` fix.. 
Also, It seem the dialog works as expected when dismissing it using swipe (on master) 
Maybe we should sync about it..

Reply:
Did you test it in public or private?

Reply:
hmm wait.. 
Wasn't the issue that after dismissing using swipe the Dialog doesn't show the next time?

Reply:
ActionSheet (private)

Reply:
So shouldn't the fix be part of ActionSheet code?

Reply:
No, it's the same as the previous PR to the Dialog for RN63, it fixes the `onDialogDismissed` API